README.md

```diff
@@ -50,6 +50,28 @@ Merge a PR by branch
 Delete a PR
 
     github-pr delete -r dataxu/test_repo -n 17
+    
+Check conditional status checks
+    !!!This check only looks at comments AFTER the latest commit, to validate that the
+    most recent code (most recent git sha pushed to the PR) has been peer reviewed!!!
+    
+    This will check to make sure that the owner can not apply a shipable comment on their own code    
+        
+        github-pr check-condition -r dataxu/dcommand -n 84 --condition-non-owner-merger
+    
+    This takes the path to the MAINTAINERS file inside the repo
```

Comment:
What do you think of `This takes the local path to a file containing github usernames, one per line, and checks to make sure the shipit comment came from one of these usernames who is an approved merger`? I think `inside the repo` is not correct, and it doesn't have to be the MAINTAINERS.txt file, that is just our convention - it is just any text file locally